docs/gallery.rst

```diff
@@ -76,3 +76,5 @@ Gallery
 .. image:: _static/img/viz_thumbnails/horizon.png
    :scale: 25 %
 
+.. image:: _static/img/viz_thumbnails/mapbox.png
```

Comment:
the file you added is 1336 px wide initially, but does it need to be that big? also it's almost 1mb. you can use a free tool called image optim (https://imageoptim.com/mac) to get the kb size down about ~28%. it would get even smaller if we could resize the image to the needed size as well.